core/src/test/scala/kafka/utils/TestInfoUtils.scala

```diff
@@ -67,4 +67,22 @@ object TestInfoUtils {
     else
       None
   }
+
+  /**
+   * Returns whether transaction version 2 is enabled.
+   * When no parameter is provided, the default returned is true.
+   */
+  def isTransactionV2Enabled(testInfo: TestInfo): Boolean = {
+    if (testInfo.getDisplayName.contains("isTV2Enabled=")) {
```

Comment:
Ohh I followed the kraftQuorum style, mainly so we could set a default for the tests, but since the default version can is set in the formatter as well I guess we don't need it, I can actually do what you suggested! Thanksss!

Reply:
Can we make this the one liner? It would still have a default as false.

Reply:
Set default to true!